libs/utils/android/workload.py

```diff
@@ -132,6 +130,9 @@ def tracingStop(self):
             else:
                 self._log.info('Waiting systrace report [%s]...',
                                  self.trace_file)
+                if (self._trace_time is None):
+                    # Systrace expects <enter>
+                    self._systrace_output.sendline("")
```

Comment:
Not big deal since we are both ' and "... but perhaps we can try to be consistent at lest within functions and, thanks to your previous removal, that's now the only usage or "... let's switch to '